fecgov/openFEC
# add integrated test for /candidates/totals/

## Summary (required)
add integrated test(s) to aid in debugging `candidates/totals/` where number of candidates is not consistent across various access methods (related issue #3673)
- Resolves #3791

_Include a summary of proposed changes._
add integrated tests to `openFEC/tests/integration/test_integrated.py`

## How to test the changes locally
- `git checkout feature/3791-write-candidates-totals-test-case`
- `cd openFEC/`
- `pytest` (connect to `pg` if not already)--all tests should pass.

## Impacted areas of the application
List general components of the application that this PR will affect:
-  mainly tests, but work done in `test_integration.py` enables additional programmatic testing of other endpoints by importing similar functionality as `test_amendment_chain.py`

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
feature/other_pr | [[WIP] Update house candidate datatable to show full election results #2917](#2917)
